### PR TITLE
Fix wrong validation set

### PR DESCRIPTION
**Pull request by gersakbogdan**
Validation loss and accuracy should be tested against mnist.validation set.

### CODE REVIEW

**Comment by Brok-Bucholtz** [comment]
This was fixed in commit ad526f261439283cd1621ce0d996c448b58c64d9
